src/main/webapp/components/Wrapper.js

```diff
@@ -0,0 +1,19 @@
+import * as React from "react";
+import {getField} from "utils/getField";
+
+/**
+  * Container element used to wrap DOM elements.
+  * Used as a abstraction for "div" HTML element.
+  * 
+  * @param props an object specifying the child components
+  * and optionally specifying a className.
+  *
+*/
+export const Wrapper = (props) => {
```

Comment:
At a high level this abstraction doesn't hold its own weight: it's too generic and it's only used to set a class name. I'd suggest to directly use a <div> (or whatever's appropriate) in the concrete child components, also setting a concrete className there.
On a separate note, I'd rather define more strict conditions on props so that the code doesn't have to do default-initializations in general. Say for this component (ignoring that I think we don't need it), I'd document that props.className and props.children are required fields with their intended meanings and not use getField in that case at all.
Note that inlining this into concrete components has additional advantages, as it will allow you to specify the class name directly instead of having it part of the props dictionary. Use props to propagate data fields and try to keep style-related information out of it.

Reply:
+1 for the insight. Based on this suggestion, I will close this PR and revert to div until the purpose of this component because more precise.